homeassistant/components/youtube/sensor.py

```diff
@@ -58,6 +59,15 @@ class YouTubeSensorEntityDescription(SensorEntityDescription):
         entity_picture_fn=lambda channel: channel[ATTR_ICON],
         attributes_fn=None,
     ),
+    YouTubeSensorEntityDescription(
+        key="views",
+        translation_key="views",
+        native_unit_of_measurement="views",
+        available_fn=lambda _: True,
+        value_fn=lambda channel: channel[ATTR_TOTAL_VIEWS],
+        entity_picture_fn=lambda channel: channel[ATTR_ICON],
```

Comment:
Should we just do an icon here instead of the channel icon? I think it can look strange if you add more sensors that they all have the same picture.

Wdyt?

Reply:
That's true, although it doesn't bother me. Not sure if there is an appropriate icon for both of them.

Reply:
I think subscribers is fine as is, but let's add one for the views, not sure if there is a viable icon for that. Please use the icon translations to add one

Reply:
I'm wondering, what makes the `views` any different from the `subscribers`?